backend/sublet/serializers.py

```diff
@@ -162,13 +169,19 @@ class Meta:
             "images",
         ]
 
```

Comment:
Is there any chance serializers are not returning dictionaries?

Reply:
if this is about the `to_representation` thing, it seems to be built into rest_framework to convert